### PR TITLE
minor typos and formatting fixes in documentation

### PR DESCRIPTION
**Pull request by stragu**
Apologies for not testing building the package before submitting the pull request, but given that this only touches the function documentation text, I did not think it would be necessary.

### CODE REVIEW

File: R/fortify-map.r [hunk 1]
```diff
@@ -39,16 +39,16 @@ fortify.map <- function(model, data, ...) {
...
 #'
-#' @param map name of map provided by the \pkg{maps} package.  These
+#' @param map name of map provided by the \pkg{maps} package. These
 #'   include [maps::county()], [maps::france()],
 #'   [maps::italy()], [maps::nz()],
 #'   [maps::state()], [maps::usa()],
 #'   [maps::world()], [maps::world2()].
-#' @param region name of subregions to include.  Defaults to `.` which
-#'   includes all subregion.  See documentation for [maps::map()]
+#' @param region name of subregion(s) to include. Defaults to `.` which
```
Comment (yutannihilation) [inline R/fortify-map.r:50]: For consistency, "name(s)"?

e.g. 
https://github.com/tidyverse/ggplot2/blob/acaa98e4cfcd922a737f35c16ec3dd6079aab3dc/R/scale-hue.r#L9

Comment (stragu) [reply R/fortify-map.r]: Thanks @yutannihilation , that makes sense. I amended the commit to include your suggestion.

**Review by yutannihilation — APPROVED** [review]

**Comment by yutannihilation** [comment]
Thanks!

**Pull request merged**